plex/provider_observer.py

```diff
@@ -377,8 +377,10 @@ def _ProcessChangedPlexItems(self, changedPlexItems: List[tuple]):
         changedItems = []
         for (changesetType, plexItemId, plexItemClass) in changedPlexItems:
             item = None
-            if changesetType in \
-               (xbmcmediaimport.MediaImportChangesetTypeAdded, xbmcmediaimport.MediaImportChangesetTypeChanged):
+            if (
+                    changesetType == xbmcmediaimport.MediaImportChangesetTypeAdded
+                    or changesetType == xbmcmediaimport.MediaImportChangesetTypeChanged
+            ):
```

Comment:
Why did you change this? `pylint` and `flake8` both usually propose to use `x in (foo, bar)` instead of `x == foo or x == bar`.

Reply:
Oh, I just saw the `\` and merged over-top. Pushing bringing in another commit now to set it to

```
            if changesetType in (
                    xbmcmediaimport.MediaImportChangesetTypeAdded,
                    xbmcmediaimport.MediaImportChangesetTypeChanged
            ):
```

Reply:
Updated